docs/usage/customization-guide.md

```diff
@@ -32,7 +32,6 @@ labeling:
 
 ## NodeFeature custom resource
 
-**EXPERIMENTAL**
```

Comment:
Also here I think the babbling about `-enable-nodefeature-api` should be reworded, or perhaps removed completely. WDYT?